content/Video_Display.md

```diff
@@ -290,46 +290,46 @@ of this brightness correction.
 
 Writing to this register launches a DMA transfer from ROM or RAM to OAM
 memory (sprite attribute table). The written value specifies the
-transfer source address divided by 100h, that is, source & destination are:
+transfer source address divided by $100, that is, source and destination are:
 
 ```
-Source:      XX00-XX9F   ;XX in range from 00-F1h
-Destination: FE00-FE9F
+Source:      $XX00-$XX9F   ;XX = $00 to $F1
+Destination: $FE00-$FE9F
 ```
 
 The transfer takes 160 machine cycles: 152 microseconds in normal speed
 or 76 microseconds in CGB Double Speed Mode. On DMG, during this time,
-the CPU can access only HRAM (memory at FF80-FFFE); on CGB, the bus used
+the CPU can access only HRAM (memory at $FF80-$FFFE); on CGB, the bus used
 by the source area cannot be used (this isn't understood well at the
-moment, it's recommended to assume same behavior as DMG). For this
+moment; it's recommended to assume same behavior as DMG). For this
 reason, the programmer must copy a short procedure into HRAM, and use
 this procedure to start the transfer from inside HRAM, and wait until
 the transfer has finished:
 
 ```
  run_dma:
-  ld a, start address / 100h
-  ldh  (FF46h),a ;start DMA transfer (starts right after instruction)
-  ld  a,28h      ;delay...
+  ld a, start address / $100
+  ldh  ($FF46),a ;start DMA transfer (starts right after instruction)
+  ld  a,$28      ;delay...
  wait:           ;total 4x40 cycles, approx 160 μs
   dec a          ;1 cycle
   jr  nz,wait    ;3 cycles
   ret
 ```
 
 Because sprites are not displayed while OAM DMA is in progress, most
-programs execute this procedure from inside of their VBlank
-procedure. But it is also possible to execute it during display redraw
-also, allowing to display more than 40 sprites on the screen (that is, for
-example 40 sprites in upper half, and other 40 sprites in lower half of
+programs execute this procedure from inside of their V-Blank
+handler. But it is also possible to execute it during display redraw,
```

Comment:
I think it should be clarified that "display redraw" means Mode 2 and 3; further, while it's in progress, the PPU reads all of OAM as $FF, possibly causing graphical errors if started during Mode 3 as a sprite is about to be drawn.

Reply:
Are the graphical errors different if the DMA transfer starts while in Mode 2 instead of in Mode 3?

Reply:
If it starts during Mode 3 as a sprite is about to be drawn, the tile and/or attr may be read as $FF, potentially producing a small graphical error. Apparently, the PPU re-reads Y and X (as $FF), and so normally doesn't display OBJ anymore. I kinda wonder why, but it seems that it's how it is.

Reply:
Updated